Makefile

```diff
@@ -19,13 +19,18 @@ MODE ?= release
 # ARCH = 64bit or 32bit
 UNAME_S = $(shell uname -s)
 UNAME_P = $(shell uname -p)
+UNAME_M = $(shell uname -m)
 ifeq ($(UNAME_S), Darwin)
 SYSTEM ?= macosx
 ARCH ?= 64bit
 FRAMEWORK ?= $(shell which sdl2-config 1>/dev/null && echo "" || echo "framework")
 else ifeq ($(UNAME_S), Linux)
 SYSTEM ?= linux
-ARCH ?= 64bit
+	ifeq ($(UNAME_M), aarch64)
+		ARCH ?= arm64
```

Comment:
Do we still need to detect the architecture?

Reply:
To disable the glibc_compat on arm64 I do until a better solution is found. I haven't looked into an alternative yet.

Reply:
Ah I see. No worries then :)